debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -374,18 +385,28 @@ protected void handleServerHeartbeat(Event event) {
     /**
      * Handle the supplied event that signals that an out of the ordinary event that occurred on the master. It notifies the slave
      * that something happened on the master that might cause data to be in an inconsistent state.
-     * 
+     *
      * @param event the server stopped event to be processed; may not be null
      */
     protected void handleServerIncident(Event event) {
-        logger.trace("Server incident: {}", event);
+        if (event.getData() instanceof EventDataDeserializationExceptionData) {
+            EventDataDeserializationExceptionData data = event.getData();
+
+            logger.error(
+                    String.format("Exception during event data deserialization (source info: %s, partion: %s)", source, source.partition()),
+                    data.getCause()
```

Comment:
In fact, it'd be great if we actually wrote out what the user should do to see more info about this event, such as running something like the following to see this particular event:
```
mysqlbinlog --start-position=1200 --stop-position=1201 --verbose binlog.0000003
```
where the start position is access from `source.restartBinlogPosition()`, stop position is obtained from `source.nextBinlogPosition()` that needs to be added (and should return `source.currentBinlogPosition + source.currentEventLengthInBytes`), and the binlog filename from `source.binlogFilename()`. I think these are correct, since they are set [here](https://github.com/gunnarmorling/debezium/blob/6a8add739f77a43ea18659c1e992cc8a11d4344d/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java#L324) before the [event is handled](https://github.com/gunnarmorling/debezium/blob/6a8add739f77a43ea18659c1e992cc8a11d4344d/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java#L330) and thus before we might process this exception.